.buildkite/pipeline.yml

```diff
@@ -16,7 +16,7 @@ steps:
       - n0izn0iz/docker#v3.5.4:
           image: bertytech/protoc:22
           propagate-uid-gid: true
-          workdir: /go/src/berty.tech/berty
+          workdir: /go/src/berty.tech/berty/v2
```

Comment:
The way gomodules are working for major version >=2 is not exactly mapping the filesystem, in our case,  you should not have v2 in your path, but only in the "go.mod" file and in the go imports

Reply:
thanks for the explanations